packages/@lwc/engine/src/framework/upgrade.ts

```diff
@@ -4,74 +4,12 @@
  * SPDX-License-Identifier: MIT
  * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
  */
-import {
-    assert,
-    assign,
-    isFunction,
-    isNull,
-    isObject,
-    isUndefined,
-    toString,
-    HiddenField,
-    createHiddenField,
-    getHiddenField,
-    setHiddenField,
-} from '@lwc/shared';
-import {
-    createVM,
-    removeRootVM,
-    appendRootVM,
-    getAssociatedVM,
-    VMState,
-    getAssociatedVMIfPresent,
-} from './vm';
+import { isFunction, isNull, isObject, toString } from '@lwc/shared';
```

Comment:
Let's rename this file as well, it has nothing to do with upgrade anymore.

Reply:
maybe `create-element.ts`